build.gradle

```diff
@@ -277,7 +277,7 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.1'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.2_CCD-3535'
```

Comment:
This needs to be updated to a full release version of the hmcts/ccd-test-defintions before going into master. see https://github.com/hmcts/ccd-test-definitions/pull/50.

Reply:
Resolved in 6969fc3f64282b8a73ccbd9991759ca861962bb7

Reply:
Done in [6969fc3](https://github.com/hmcts/ccd-data-store-api/pull/2058/commits/6969fc3f64282b8a73ccbd9991759ca861962bb7)